scripts/db_migrator.py

```diff
@@ -180,9 +180,10 @@ def migrate_feature_table(self):
         '''
         feature_table = self.configDB.get_table('FEATURE')
         for feature, config in feature_table.items():
-            state = config.pop('status', 'disabled')
-            config['state'] = state
-            self.configDB.set_entry('FEATURE', feature, config)
+            if 'status' in config:
+                state = config.pop('status', 'disabled')
```

Comment:
Can we fix this bug by change default value to `enabled`?

Reply:
I believe we can remove the default value. The if check will make sure that 'status' is always present. Let me make the change